### PR TITLE
curvefs-tool delete-fs

### LINKED ISSUE #681: curvefs-tool delete-fs
**Is your feature request related to a problem? (你需要的功能是否与某个问题有关?)**

Use curvefs-tool to delete fs

**Describe the solution you'd like (描述你期望的解决方法)**

curvefs-tool delete-fs [-confPath=/etc/curvefs/tools.conf] -fsname=curvefs -confirm=0 [-mdsAddr=127.0.0.1:16700,127.0.0.1:26700]

**Describe alternatives you've considered (描述你想到的折衷方案)**

send a DeleteFsRequest to mds

**Additional context/screenshots (更多上下文/截图)**

### PR DESCRIPTION
**Pull request by Cyber-SiKu**
### What problem does this PR solve?

Issue Number: close #681

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: curvefs/src/client/s3/client_s3_cache_manager.cpp [hunk 1]
```diff
@@ -711,16 +711,19 @@ std::vector<ObjectChunkInfo> FileCacheManager::GetReadChunks(
                     VLOG(9)
                         << "chunkTmp offset:" << chunkTmp.s3ChunkInfo.offset()
                         << ", len:" << chunkTmp.s3ChunkInfo.len()
-                        << ", objectoffset:" << chunkTmp.objectOffset;
+                        << ", objectoffset:" << chunkTmp.objectOffset
+                        << ",j:"  << j;
```
Comment (aspirer) [inline curvefs/src/client/s3/client_s3_cache_manager.cpp:715]: These are not your codes? remove them pls.

File: curvefs/src/client/s3/client_s3_cache_manager.cpp [hunk 2]
```diff
@@ -711,16 +711,19 @@ std::vector<ObjectChunkInfo> FileCacheManager::GetReadChunks(
...
                 waitingDel.push_back(j);
             }
         }
 
         std::vector<int>::iterator iter = waitingDel.begin();
+        int m = 0;
         for (; iter != waitingDel.end(); iter++) {
-            chunks.erase(chunks.begin() + *iter);
+            chunks.erase(chunks.begin() + (*iter - m));
+            m++;
```
Comment (aspirer) [inline curvefs/src/client/s3/client_s3_cache_manager.cpp:726]: ditto

File: curvefs/src/client/s3/client_s3_cache_manager.cpp [hunk 3]
```diff
@@ -729,6 +732,14 @@ std::vector<ObjectChunkInfo> FileCacheManager::GetReadChunks(
...
         chunkTmp.objectOffset = tmp.offset() % blockSize;
         chunks.push_back(chunkTmp);
+        /*
+        auto iter1 = chunks.begin();
+        for (; iter1 != chunks.end(); iter1++) {
+        VLOG(9) << "GetReadChunks chunkid:" << iter1->s3ChunkInfo.chunkid()
+                << ",offset:" << iter1->s3ChunkInfo.offset()
+                << ",len:" << iter1->s3ChunkInfo.len()
+                << ",objectoffset:" << iter1->objectOffset;
+        }*/
```
Comment (aspirer) [inline curvefs/src/client/s3/client_s3_cache_manager.cpp:742]: ditto

Comment (aspirer) [reply curvefs/src/client/s3/client_s3_cache_manager.cpp:742]: and below.

File: curvefs/src/client/s3/client_s3_cache_manager.cpp (hunk 1 above)
Comment (Cyber-SiKu) [reply curvefs/src/client/s3/client_s3_cache_manager.cpp]: > These are not your codes? remove them pls.

fix

File: curvefs/src/client/s3/client_s3_cache_manager.cpp (hunk 2 above)
Comment (Cyber-SiKu) [reply curvefs/src/client/s3/client_s3_cache_manager.cpp]: > ditto

fix

File: curvefs/src/client/s3/client_s3_cache_manager.cpp (hunk 3 above)
Comment (Cyber-SiKu) [reply curvefs/src/client/s3/client_s3_cache_manager.cpp]: > ditto

fix

**Pull request closed without merging**